src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -357,27 +357,13 @@ private static class Logging {
      */
     private boolean streamFilterSet;
 
-    /**
-     * cache LUDCL (Latest User Defined Class Loader) till completion of
-     * read* requests
-     */
-
-    @SuppressWarnings("removal")
-    private static final boolean isClassCachingEnabled =
-            AccessController.doPrivileged(new GetClassCachingSettingAction());
+    /* cache LUDCL (Latest User Defined Class Loader) till completion of read requests */
+    /* if true LUDCL/forName results would be cached, true by default starting Java8 */
```

Comment:
Since you're updating these comments, please start each sentence with a capital and end with a period.
Please also use "until" instead of "till".
```java
    /* Cache LUDCL (Latest User-Defined Class Loader) until completion of read requests. */
    /** If true LUDCL/forName results would be cached, true by default starting Java 8. */
```

Reply:
Updated